hk-package-clean-except/action.yml

```diff
@@ -0,0 +1,95 @@
+name: "Clean all package versions except certain tags."
+description: "Action cleaning up all package versions published at ghcr.io except for certain tags."
+
+inputs:
+  package-name: 
+    description: "Package name."
+    required: true
+    type: string
+  token:
+    description: "Token with package deletion permissions."
+    required: true
+    type: string
+  tags-kept:
+    description: "Tags to be kept. Pass them as a list: e.g. 'latest, latest-unstable'."
+    required: true
+    type: string
+  package-org:
+    description: "Organization at which packages are published."
+    required: false
+    default: 'pyansys'
+    type: string
+  python-version:
+    description: 'Desired Python version.'
+    default: '3.10'
+    required: false
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Set up Python ${{ inputs.python-version }}"
+      uses: pyansys/actions/_setup-python@main
+      with:
+        python-version: ${{ inputs.python-version }}
+        use-cache: false
+
+    - name: "Install ghapi"
+      run: |
+        python -m pip install --upgrade pip
+        pip install ghapi
+
+    - name: "Clone script"
+      run: |
+        echo ''                                                                            >> clean-except.py

```

Comment:
It may be possible to play with the EOL parameter. Also, I think it is possible to specify the console as "python" to run directly python code.

Reply:
For example, somthing like:

```console
cat > script.py <<'EOF'  

import os

from ghapi.all import GhApi
from ghapi.core import print_summary
...

EOF
```

Reply:
Ahhh! Let me try that. It sounds really interesting :)

Reply:
Done!